panther_analysis_tool/core/analysis_cache.py

```diff
@@ -441,6 +442,9 @@ def _populate_sqlite(
     if spec.analysis_type() == AnalysisTypes.PACK:
         return
 
+    if spec.is_experimental():
+        return
+
     if id_value not in versions:
```

Comment:
nit: why not retrieve the id_value after the early return checks?

Reply:
[move id value to when it is needed](https://github.com/panther-labs/panther_analysis_tool/pull/719/commits/69a249b06f5d068dc3258805596af8cf840367f3)